sharding-core/sharding-core-rewrite/src/main/java/org/apache/shardingsphere/core/rewrite/feature/encrypt/token/pojo/EncryptPredicateRightValueToken.java

```diff
@@ -72,7 +72,6 @@ private String toStringForEqual() {
     
     private String toStringForIn() {
         StringBuilder result = new StringBuilder();
-        result.append("(");
```

Comment:
Can we fix this bug by modifying the startIndex?  By reviewing the unit test, personally it seems wired that `toStringForIn()` returns `'a', 'b')`, and `('a', 'b')` is complete string item. How about you?